### ISSUE TITLE
pd.DataFrame supported APIs (.merge)?

### ISSUE THREAD

**Issue by pharoahe34** [modin-project/modin#2472]
Hi,

Following my email to feature_requests@modin.org, I open an issue here on github as Devin adviced me.
After the `print('merge progress')` in the python script bellow I use the merge method from pandas with an argument called `indicator`. This last one add a '_merge' column that I find really useful when dealling with data.
Is there any chance that you will implement that ?
Thanks for all work done amazing tools.

```python        
# transform and read csv file from mnt/lambda/temp into dataframes
dfrefe = pd.read_csv(localfilerefe, encoding='utf-8', delimiter=sep)
dftest = pd.read_csv(pathlocalfile, encoding='utf-8', delimiter=sep)
# compare dftest_full vs dfrefe
print('merge progress')
comparison_df = dfrefe.merge(dftest, indicator=True, how='outer')
# remove value present in both df
df_changes = comparison_df[comparison_df._merge != 'both']
# finding update rows
df_potential_update = df_changes[df_changes.duplicated(subset=cle, keep=False)]
```

**Comment by YarShev** [comment]
Hi @pharoahe34 , thanks for posting! Currently we have a parallel implementation of `merge` method for `left` and `inner` values of `how` parameter. Are you interested in parallel implementation for other values of `how` parameter (`right` and `outer`)?

**Comment by pharoahe34** [comment]
Hi @YarShev 

Yes for sure thats could make sense for me.
 I will be to able to retrieve data more easily ^^
Thanks

**Comment by YarShev** [comment]
@pharoahe34 , we already have similar issue #656. We will try to consider it in nearest releases.

**Comment by mvashishtha** [comment]
Duplicate of #656
